pages/components/shared/TechStackAutoComplete.tsx

```diff
@@ -21,23 +21,25 @@ const TechStackAutoComplete: React.FC<Props> = ({ projectTechStacks, setProjectT
             id="arrival-date"
             getOptionLabel={option => option.name}
             options={techstack}
+            isOptionEqualToValue={(o, v) => o.name === v.name}
```

Comment:
only this line was added; now selected options cannot be added during an edit and there are no warnings in the console.

sorry for the multiple code style changes below; I think it's prettier acting up. :/

Reply:
That's fine prettier always do this.